sambahsa.md

```diff
@@ -2279,7 +2280,7 @@ in a problematic situation []{#anchor-165}**{ correct Ѵ}**
 []{#anchor-166}**Un pateric amor –** []{#anchor-167}a paternal love
 []{#anchor-168}**{ wrong X}**
 
-[]{#anchor-169}**Un patruw amor –** a paternal love []{#anchor-170}**{
+[]{#anchor-169}**Un patruw amor –** a paternal love (attachment) []{#anchor-170}**{
```

Comment:
Just a precision you couldn't know yet. I have *discovered* that the Romance word "amor" has a PIE ancestor meaning "to be bound / attached to" (cf. Greek : omnumi = I swear"). So, the real first meaning of "amor" is "attachment"  ("emotional connection between individuals")